pkgs/by-name/mi/mirrord-bin/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  testers,
+  mirrord-bin,
+}:
+
+let
+  manifest = lib.importJSON ./manifest.json;
+in
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Reply:
"Although I don't know the specific reason, stdenv is required here. See `buildInputs`.